front/lib/actions/server.ts

```diff
@@ -204,7 +209,12 @@ export async function runAction(
     logger
   );
 
-  const res = await api.runApp(action.app, config, inputs);
+  const res = await api.runApp(
+    auth.getNonNullableUser(),
```

Comment:
this will throw, which I don't think we want ?

Reply:
Updated.